examples/sorry-page/sorry.html

```diff
@@ -0,0 +1,1001 @@
+                class="ecl-icon ecl-icon--m ecl-u-type-color-blue-100"
+              >
+                <use
+                  xlink:href="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v2.38.0/eu-preset-website/images/icons/sprites/icons.svg#general--language"
+                ></use>
+              </svg>
+              <span
+                style="
+                  left: 50%;
+                  position: absolute;
+                  top: 0;
+                  transform: translate(-50%, 3px);
+                "
+                class="ecl-u-type-xs ecl-u-type-color-white-100 ecl-u-type-uppercase"
+                >sl</span
+              >
+            </span>
+            <div class="ecl-u-ml-m ecl-u-width-100">
+              <div
+                class="ecl-u-type-prolonged-m ecl-u-type-color-blue-100 ecl-u-type-bold"
+              >
+                Strežnik je začasno nedosegljiv
+              </div>
+              <p
+                class="ecl-u-type-paragraph ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-mt-s ecl-u-mb-none ecl-u-pb-m"
+              >
+                Opravičujemo se za morebitne nevšečnosti. Poskusite ponovno
+                pozneje.
+              </p>
+            </div>
+          </div>
+
+          <div
+            class="ecl-u-d-inline-flex ecl-u-pt-m ecl-u-mt-l ecl-u-width-100"
+          >
+            <span class="ecl-u-d-flex" style="position: relative">
+              <svg
+                focusable="false"
+                aria-hidden="true"
+                class="ecl-icon ecl-icon--m ecl-u-type-color-blue-100"
+              >
+                <use
+                  xlink:href="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v2.38.0/eu-preset-website/images/icons/sprites/icons.svg#general--language"
+                ></use>
+              </svg>
+              <span
+                style="
+                  left: 50%;
+                  position: absolute;
+                  top: 0;
+                  transform: translate(-50%, 3px);
+                "
+                class="ecl-u-type-xs ecl-u-type-color-white-100 ecl-u-type-uppercase"
+                >sv</span
+              >
+            </span>
+            <div class="ecl-u-ml-m ecl-u-width-100">
+              <div
+                class="ecl-u-type-prolonged-m ecl-u-type-color-blue-100 ecl-u-type-bold"
+              >
+                Servern är inte tillgänglig för tillfället
+              </div>
+              <p
+                class="ecl-u-type-paragraph ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-mt-s ecl-u-mb-none ecl-u-pb-m"
+              >
+                Vi ber om ursäkt för besväret. Försök igen senare.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="ecl-u-bg-blue-100 ecl-u-pa-l"></footer>
+
+    <script
+      src="https://cdn1.fpfis.tech.ec.europa.eu/ecl/v2.38.0/eu-preset-website/scripts/ecl-eu-preset-website.js"
```

Comment:
yet another question? Is it really necessary to have the ECL preset JS loaded for this page, as well as `autoInit`? `svgforeverybody` isn't it enough?

Reply:
fair point, I'll remove it